src/components/views/RoleView/RoleView.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import Box from 'components/Box';
+import { Role, RoleData } from 'modules/roles/types';
+import { UserData } from 'modules/users/types';
+import { useCollectionData } from 'modules/data';
+import { ROLES } from 'utils/collectionNames';
+
+const RoleView = (props: RoleViewProps) => {
+  const { role, users } = props;
+  const { removeDoc } = useCollectionData<Role, RoleData>(ROLES);
+  const [remainingUsers, setRemainingUsers] = useState([]);
+  return (
+    <Box>
+      <Box style={{ display: 'flex', flexDirection: 'row' }}>
```

Comment:
```suggestion
      <Box display="flex" flexDirection="row">
```

This is also styled-system. Doesn't really matter since we doing over the UI but it's a better dev experience for u too.